modules/claims_api/spec/lib/claims_api/v2/poa_pdf_constructor/individual_spec.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'claims_api/v2/poa_pdf_constructor/individual'
+require_relative '../../../../support/pdf_matcher'
+
+describe ClaimsApi::V2::PoaPdfConstructor::Individual do
+  let(:temp) { create(:power_of_attorney, :with_full_headers) }
+
+  before do
+    Timecop.freeze(Time.zone.parse('2020-01-01T08:00:00Z'))
+    temp.form_data = {
+      representative: {
```

Comment:
This and the organization spec aren't the most filled out the pdf can be. Not sure if the team has thoughts on how robust the example request body should be.

Reply:
I think if it makes sense to do a full and min. example (so 2) we can do that.  But personally I do think it makes sense to have them as full as they can be.  Not sure if anyone else has strong feelings on it.

Reply:
I would agree. I'd rather err on the side of caution. More tested is probably best. If we have _all_ (max / full) in one spec, do we really need a test for min? What's the logic behind adding min as a secondary check? Shouldn't max catch all the min and more?

Reply:
I would agree a min example is not needed, probably shouldn't have even mentioned it, just trying to stress that if a small example was useful for some reason I still think prioritizing a max example, for the reasons you outlined, is best

Reply:
I think a max example is good. I went more minimal to align with V1 and I also wasn't sure how long lived my max version would be with changes coming to address and non-veteran claimants.

More than happy to put together a max example.

Reply:
Update both tests to use the full schema attributes for their respective endpoints